### ISSUE TITLE
Video recording sound problem

### ISSUE THREAD

**Issue by tonai** [slidevjs/slidev#602]
**Describe the bug**
When I make multiple records during the same session (because my computer crash when the record is too long).
Then the second recording does not have any sound (only the first recording contains the sound).

**To Reproduce**
1. Start serveur
2. Start recording
3. Speak
4. Stop recording
5. Start recording
6. Speak
7. Stop recording
8. Look at records

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04
 - Browser: 101.0.4951.64 
 - Slidev version: 0.32.1

**Comment by michael-azogu** [comment]
sounds like memory problem. seems like having separate node processes would fix this

**Comment by tonai** [comment]
In fact it is not so reproductible, because I tried with a short record and it is ok.
But it happens two times when I tried to record a real session.
In both cases the first record is about 1h long.

It is not related to node, because everything happen in the browser.
I think, the problem is more from RecordRTC that saves all the record in memory.
So yes it is probably related to memory.
RecordRTC doe not provide a way to stream the video (related issue https://github.com/muaz-khan/RecordRTC/issues/778).
